### ISSUE TITLE
Filename with $$

### ISSUE THREAD

**Issue by danilobellini** [IgnorantGuru/rmdupe#3]
I've had a problem with a filename ending with `$$A`, which had the `$$` replaced by a number (here it was `5307`) when calling `rm`:

```
    removing /media/danilo/[...]/Update.$$A
rm: cannot remove ‘/media/danilo/[...]/Update.5307A’: No such file or directory
      rmdupe: Error: removal failed of copy /media/danilo/[...]/Update.$$A
```

**Comment by danilobellini** [comment]
The same happens with other names that have a `$` followed by something that could be a "variable name". As a workaround I'm renaming them to keep another symbol before using _rmdupe_:

```
find -depth -name '*$*' | while read fname ; do mv "$fname" "$(echo $fname | tr $ _)" ; done
```

But the reference file doesn't need to be renamed, just the one that would be removed.

**Comment by IgnorantGuru** [comment]
Thanks - these filename issues should now be corrected in 1.0.4.
